### PR TITLE
created landing page of watchcart

### PR DESCRIPTION
**Pull request by agrawalshanu708**
created landing page of wtachcart(e-commerce)
added a slider banner
added few cards according to category
add list of brands
wc-lp-1
wc-lp-2
wc-lp-3

### CODE REVIEW

File: landing-page/style.css [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  @keyframes fade {
+    from {opacity: .4} 
+    to {opacity: 1}
+  }
+  /* ----------------------------------dots */
+
+  .dot {
+    cursor: pointer;
+    height: 15px;
+    width: 15px;
```
Comment (yashpatel1797) [inline landing-page/style.css:52]: use 'rem' here as well.

File: landing-page/style.css [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    margin: 0 2px;
+    background-color: #bbb;
+    border-radius: 50%;
+    display: inline-block;
+    transition: background-color 0.6s ease;
+    margin-bottom: 8rem;
+  }
+  .active, .dot:hover {
+    background-color: #717171;
+  }
```
Comment (yashpatel1797) [inline landing-page/style.css:62]: for background colour declare a variable for this also.

**Review by zabihshaik — APPROVED** [review]
Everything looks good.

**Review by yashpatel1797 — APPROVED** [review]
for color you can declare variables for every color so it would be consistent. everything looks good.

File: landing-page/style.css (hunk 1 above)
Comment (agrawalshanu708) [reply landing-page/style.css:52]: sure

**Review by therajatg — APPROVED** [review]
Looks good

File: landing-page/style.css (hunk 2 above)
Comment (agrawalshanu708) [reply landing-page/style.css:62]: sure

File: resetcss.css [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+  background-color: var(--primary-color) ;
+  color: var(--background-color);
+  border-radius: 1rem;
+  margin-top: 1rem;
+  cursor: pointer;
+}
+/* ------------------------------------------------------------------------------ */
+
+button:hover{
+  background-color: #0369a1;
```
Comment (yaswanthmyneni) [inline resetcss.css:195]: changes: use CSS variable for all the colors.

File: resetcss.css [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+ 
+a{
+      text-decoration: none;
+      font-size: 3rem;
+  }
+p{
+    font-family:'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+
+      font-size: 1.7rem;
+      margin:  1rem 0 0 0;
```
Comment (yaswanthmyneni) [inline resetcss.css:35]: suggestion: you can use margin-top here.

**Review by yaswanthmyneni — APPROVED** [review]
You can merge after changes.

**Pull request merged**